app/Models/User.php

```diff
@@ -119,6 +119,7 @@ class User extends Authenticatable implements HasLocalePreference, FilamentUser,
     protected $hidden = [
         'remember_token',
         'password',
+        'password_history',
```

Comment:
Let's also add `password_last_updated_at` to `hidden`. As this is technically sensitive data we would not want exposed.